packages/block-library/src/button/style.scss

```diff
@@ -49,3 +49,14 @@ $blocks-button__height: 56px;
 		border: 2px solid;
 	}
 }
+
+.editor-format-toolbar__url-inline {
+	margin-top: 16px;
+	color: $dark-gray-500;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+	font-size: 13px;
+
+	.block-editor-url-input {
```

Comment:
Pretty much all the styles defined here should be removed, apart from `margin-top` and `color`, both of which need to be kept. It might be nice to replace the `16px` for margin-top with a variable - `margin-top: $grid-size-large;` has the same effect.

Once all of those styles have been updated, it doesn't look too bad, but the input is missing a border and border-radius. Unfortunately that's `URLInput`'s fault again. It gets rid of them here:
https://github.com/WordPress/gutenberg/blob/5d14c988e4239baafebb35600f2f56e8a6a18952/packages/block-editor/src/components/url-input/style.scss#L18-L19

I think the right thing to do would be to add a modifier class for URLInput like `.has-border` that reintroduces them.

Reply:
Addressed [here](https://github.com/WordPress/gutenberg/pull/10128/commits/c3fc7ea40a465049efe2a3a96dc8fd85b679fc9e) and [here](https://github.com/WordPress/gutenberg/pull/10128/commits/53701ca76e9e59ef26550aa96b3bf7cc09f25f73).

Thank you ❤️